website/playground/static/index.html

```diff
@@ -22,6 +22,7 @@
 <script src="third-party/lint.js"></script>
 <script src="third-party/hardwrap.js"></script>
 <script src="third-party/sublime.js"></script>
+<script src="third-party/rx.min.js"></script>
```

Comment:
Rx is a huge library, this bloats the page unnecessarily. How about either using something like lodash's per-method packages or just implementing `debounce` locally?

Reply:
Yeah I can do something like that.